spell/spell-reviewer-mainnet-checklist.md

```diff
@@ -1,86 +1,112 @@
-# Mainnet Exec Spell Review Checklist [PE-TODO]
+# Mainnet Executive Spell Review Checklist
 
-## Mainnet
+## Mainnet YYYY-MM-DD
 
-Spell Actions:
+Spell Actions (Per Exec Doc):
 
-* TODO
-* TODO
+* _Read spell actions and instructions from the Exec Doc_
+* _List the actions being performed in this spell_
+
+## Development Stage
 
-## Checklist
 * [ ] Office Hours
-  * [ ] On (Collateral Onboarding, Keepers, Integrations, ...)
-  * [ ] matches exec doc
-* [ ] Exec Hash
-  * [ ] Run `make exec-hash` for current date, or `date=YYYY-MM-DD`
-    * [ ] Executive vote file name and date is correct
-    * [ ] [community](https://github.com/makerdao/community) repo commit hash corresponds to latest change
-    * [ ] Raw GitHub URL is correct
-    * [ ] Exec hash is correct
-  * [ ] `description` date in `DssSpell.sol` matches exec copy date
-* [ ] 30 Days Expiry
-* [ ] `lib`
-  * [ ] `dss-exec-lib`
-    * [ ] if submodule upgrades are present make sure `dss-exec-lib` is synced as well
-    * [ ] git submodule hash matches tag latest release version or above
-  * [ ] `dss-test`
-    * [ ] `dss-interfaces`
-      * [ ] git submodule hash matches github master commit
-    * [ ] `forge-std`
-      * [ ] git submodule hash matches github tag latest release version
-* [ ] `dss-interfaces`
-  * [ ] used in the current spell
-  * [ ] cleanup previous ones
-  * [ ] ensure only single import layout is used (e.g. `import "dss-interfaces/dss/VatAbstract.sol";`)
-* [ ] Static Interfaces
-  * [ ] ensure they match `dss-interfaces`
-  * [ ] check on-chain interface of deployed contract via `cast interfaces <contract_address>` to ensure correctness
-  * [ ] interface naming style should match with `Like` suffix (e.g. `VatLike`)
-  * [ ] ensure they only list used functions in spell code
-* [ ] Rates match
-  * [ ] Compare against [IPFS](https://ipfs.io/ipfs/QmVp4mhhbwWGTfbh2BzwQB9eiBrQBKiqcPRZCaAxNUaar6)
-  * [ ] Check manually via `make rates pct=<pct>` (e.g. pct=0.75, for 0.75%)
-  * [ ] Variable visibility declared as internal
-* [ ] Math matches
-  * [ ] Internal Precision
-    * [ ] `WAD = 10**18`
-    * [ ] `RAY = 10**27`
-    * [ ] `RAD = 10**45`
+  * [ ] ON
+  * [ ] OFF
```

Comment:
Nit: I find this change pointless as it doesn't help reviewer check anything, instead I suggest to revert it to the previous text: `On (Collateral Onboarding, Keepers, Integrations, ...)`